receiver/mongodbreceiver/documentation.md

```diff
@@ -11,14 +11,26 @@ These are the metrics available for this scraper.
 | **mongodb.cache.operations** | The number of cache operations of the instance. | {operations} | Sum(Int) | <ul> <li>type</li> </ul> |
 | **mongodb.collection.count** | The number of collections. | {collections} | Sum(Int) | <ul> <li>database</li> </ul> |
 | **mongodb.connection.count** | The number of connections. | {connections} | Sum(Int) | <ul> <li>database</li> <li>connection_type</li> </ul> |
+| **mongodb.cursor.count** | The number of open cursors maintained for clients. | {cursors} | Sum(Int) | <ul> </ul> |
+| **mongodb.cursor.timeout.count** | The number of cursors that have timed out. | {cursors} | Sum(Int) | <ul> </ul> |
 | **mongodb.data.size** | The size of the collection. Data compression does not affect this value. | By | Sum(Int) | <ul> <li>database</li> </ul> |
+| **mongodb.database.count** | The number of existing databases. | {databases} | Sum(Int) | <ul> </ul> |
+| **mongodb.document.delete.count** | The number of documents deleted. | {documents} | Sum(Int) | <ul> <li>database</li> </ul> |
+| **mongodb.document.insert.count** | The number of documents inserted. | {documents} | Sum(Int) | <ul> <li>database</li> </ul> |
+| **mongodb.document.update.count** | The number of documents updated. | {documents} | Sum(Int) | <ul> <li>database</li> </ul> |
```

Comment:
Are the above metrics meaningfully summed? if so should they be a single metric with label?

Reply:
Spoke with team and after reasoning through the unit and what they actually represent, we don’t think it makes sense to combine them.